src/MeetingsSDKAdapter/controls/index.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/prefer-default-export */
+export {default as RosterControl} from './RosterControl';
+export {default as MeetingControl} from './MeetingControl';
```

Comment:
maybe in reverse order (alphabetical)

Reply:
actually, I don't think this file is needed at all, the controls are imported directly from their own files